frontend/src/__tests__/cypress/cypress/tests/mocked/pipelines/pipelines.cy.ts

```diff
@@ -941,7 +941,7 @@ describe('Pipelines', () => {
     pipelineRow
       .getPipelineVersionRowById(initialMockPipelineVersion.pipeline_version_id)
       .findPipelineVersionLink()
-      .click();
+      .click({ force: true });
```

Comment:
With the css override for table truncation, I think we can remove this workaround.
```suggestion
      .click();
```